config/dev.exs

```diff
@@ -1,6 +1,6 @@
 import Config
 
-config :logger, level: System.get_env("ARCHETHIC_LOGGER_LEVEL", "debug") |> String.to_atom()
+config :logger, level: System.get_env("ARCHETHIC_LOGGER_LEVEL", "warning") |> String.to_atom()
```

Comment:
I don't think this change is necessary, and it will disturb habits for the local development

Reply:
reverted 👍 +1